### DISCUSSION TITLE
How to display different bubble menus depending on the content selected within the table

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [ueberdosis/tiptap#5697] [General Questions]
## Example 1

When text in a table is dragged.

A bubble menu appears where you can edit text size and change color.

![image](https://github.com/user-attachments/assets/c066af0f-c85a-42b6-9427-77a0a85d9b8e)

## Example 2

When table rows and columns are selected

Shows bubble menus related to tables, such as adding and merging tables

![image](https://github.com/user-attachments/assets/81c564dd-46e1-4701-a0a0-5f8f4c6f9118)

Is it possible to differentiate these so that an appropriate bubble menu appears?

**Comment by nperez0111** [comment]
Original https://github.com/ueberdosis/tiptap/discussions/5689#discussioncomment-10826641

**Reply by ryuhangyeong** [reply]
Sorry.
